api/src/controllers/application.js

```diff
@@ -164,6 +164,13 @@ router.post("/", passport.authenticate(["young", "referent"], { session: false,
       }
     }
 
+    // On vérifie que la candidature n'existe pas déjà en base de donnée.
+    const doublon = await ApplicationObject.findOne({ youngId: req.body.youngId, missionId: req.body.missionId });
```

Comment:
Utilise la value du joi Au lieu du body de la requete

```suggestion
    const doublon = await ApplicationObject.findOne({ youngId: value.youngId, missionId: value.missionId });
```